src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -300,4 +314,23 @@ private void addDoi(Map<String, String> hm, String val) {
             hm.put(FieldName.DOI, doi);
         }
     }
+
+    private boolean isDateTag(String s) {
+        for (String dateTag : dateTags) {
+            if (dateTag.equals(s)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    private int getDatePriority(String date) {
+        int i;
+        for (i = 0; i < dateTags.length; i++) {
```

Comment:
you can simply use  dateTags.indexOf(data) if you switch your array to the arraylist as mentioned earlier

Reply:
Yes, using the built in method is a much better idea.